src/content/docs/en/reference/api-reference.mdx

```diff
@@ -1490,18 +1696,25 @@ export function onRequest (context, next) {
 
 ### `next()`
```

Comment:
I wonder if this should be nested under `onRequest()` given it is an argument of that function?

```suggestion
#### `next()`
```

Reply:
I checked the history, it seems there is no particular reason for `next()` being at the same level. So I think we can safely move it under `onRequest`. However, it might seem strange to have only one subtitle in the section...

Reply:
> it might seem strange to have only one subtitle in the section...

I think it’s OK. There is a question about why `context` doesn’t also have a heading maybe?

Reply:
I guess it was tricky since the context object is already described above in `Endpoint contexts`. However, having two subheadings (one for `context` and another for `next()`) since seems preferable to me to avoid an isolated heading (`next()`). Maybe we could use something like:

![An example of context and next function as subheading of onRequest](https://github.com/user-attachments/assets/428aa22f-0897-4a05-9486-5a0ab2c2835c)

**Note:** I didn't check if `Since` could be applied to `context`, this is just an example.

Reply:
I like that a lot! And I don’t _think_ we need a `<Since>` for `context` as it was always available in middleware so covered by the main middleware one (and the availability of individual properties should be documented in the endpoint docs you linked to).

(I still don’t think we should display “Added in” in the heading for `next()`, but I guess that was not the point of this screenshot.)